Content.Server/Climbing/BonkSystem.cs

```diff
@@ -0,0 +1,51 @@
+using Content.Server.Interaction;
+using Content.Server.Stunnable;
+using Content.Shared.CCVar;
+using Content.Shared.Climbing;
+using Content.Shared.Damage;
+using Content.Shared.DragDrop;
+using Robust.Server.GameObjects;
+using Robust.Shared.Configuration;
+
+namespace Content.Server.Climbing
```

Comment:
Im pretty sure you can move this system to Content.Shared, also, use a file scoped namespace to reduce a level of identation